clang/test/dpct/cmake_migration/case_008/input.cmake

```diff
@@ -0,0 +1,18 @@
+find_package(CUDA)
+find_package(CUDA REQUIRED)
+find_package(CUDA ${REQUIRED_CUDA_VERSION} REQUIRED)
+#Current YAML rule comments find_package(MPI) as it changes CMAKE_CXX_COMPILER
+find_package(MPI)
+find_package(MPI REQUIRED)
+#Current YAML rule comments find_package(OpenMP) as it changes CMAKE_CXX_COMPILER
+FIND_PACKAGE(OpenMP)
+FIND_PACKAGE(OpenMP REQUIRED)
+find_package(CUDAToolkit)
+find_package(CUDAToolkit 11.4 REQUIRED)
+find_package(CUDAToolkit REQUIRED curand cufft cusparse_static cublas cusolver)
```

Comment:
I'd prefer to migrate to: 
find_package(IntelSYCL)
find_package(MKL)
find_pakcage(DPL)

Don't introduce helper function for stadard find_package().  There is overhead to main the helper function.